STLtoDATTranslator/Header Files/Point.h

```diff
@@ -6,3 +6,6 @@ class Point {
 	Point(int x=0,int y=0,int z=0);
 	~Point();
 };
```

Comment:
make sure to define data members private in the code to maintain abstraction, and also provide getter function for that members , its not only for this class do it for all the classes

Reply:
data members can be private.